main.tf

```diff
@@ -5,12 +5,59 @@ resource "aws_instance" "web" {
   tags = {
     Name = "web"
   }
+
+  metadata_options {
+    http_endpoint = "disabled"
+    http_tokens   = "required"
+  }
 }
 
 resource "aws_s3_bucket" "bucket" {
   bucket = "my-tf-test-bucket"
 
   tags = {
-    Name        = "bucket"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS).

source: accurics-stageone